src/payload.rs

```diff
@@ -407,10 +485,11 @@ mod tests {
         assert_eq!(config.max_retries, Some(5));
         assert!(config.skip_preflight);
         assert_eq!(decoded_tx.signatures, tx.signatures);
+        Ok(())
     }
 
     #[test]
-    fn test_config_conversion() {
+    fn test_config_conversion() -> Result<(), Box<dyn std::error::Error>> {
         let proto_config = TransactionConfig {
```

Comment:
Isn't unwrap good for tests. Panic is what we want. Can use `expect()` to give helpful message.